zio2-zmq/src/main/scala/zio/zmq/ZContext.scala

```diff
@@ -27,6 +27,7 @@ object ZContext {
             case e: ZMQException if e.getErrorCode == InterruptedFunction =>
               ZIO.unit
           }
+          .timeoutFail(new Exception("shutdown context timedout"))(60.seconds)
```

Comment:
I think this should be configurable, maybe `live(shutdownTimeout: Duration = 60.seconds)` and use that in the timeout.

I can take it tomorrow